README.md

```diff
@@ -96,19 +96,19 @@ repositories {
 Dependency:
 ```groovy
 dependencies {
-    compile "io.papermc:paperlib:1.0.6"
+    compileOnly "io.papermc:paperlib:1.0.6"
 }
 ```
 
 Shadow Jar and Relocate:
 ```groovy
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
-        classpath "com.github.jengelman.gradle.plugins:shadow:2.0.2"
+        classpath "com.github.jengelman.gradle.plugins:shadow:6.1.0"
     }
 }
```

Comment:
I dislike showing off the old style at all. Perhaps link to the shadow jar plugin instead of this stuff? https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow

Reply:
I wasn't sure which style is preferred. Personally I use the plugins' block over the legacy methods.

Reply:
The legacy methods are just that, legacy. The plugins block should always be preferred nowadays, especially on Gradle 7.